Author: Rene Engelhard <rene@debian.org>
Description: fix build.xml to remove nbproject usage as we don't have it.

--- a/build.xml
+++ b/build.xml
@@ -59,7 +59,6 @@
 
   <target name="default" depends="test" description="Build and Test."/>
 
-  <import file="nbproject/profiler-build-impl.xml"/>
   <target name="init" depends="-setup"/>
   <target name="compile-test-single" depends="compile-tests"/>
   <target name="compile-single" depends="compile"/>
@@ -620,7 +619,6 @@
       <zipfileset src="${dist}/src.zip"/>
       <zipfileset dir="lib" includes="junit.jar,clover.jar" prefix="lib"/>
       <zipfileset dir="." includes=".classpath,.project"/>
-      <zipfileset dir="nbproject" includes="**/*" prefix="nbproject"/>
       <zipfileset dir="${native}" includes="libffi,libffi/**/*" prefix="native"/>
     </zip>
     <!-- Sources package as required by maven -->
